frontend/src/components/Plan/hook/fetchPlanData.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+import { loadingStatus } from '../../../utils/loadingStatus'
+import axios from 'axios'
+
+export const fetchPlanData = (planIdParam: string): any => {
+  const [status, setStatus] = useState<loadingStatus>(loadingStatus.NOT_LOADED)
+  const [data, setData] = useState<any>(null)
+
+  useEffect(() => {
+    const controller = new AbortController();
+    const fetchPlanData = async (): Promise<void> => {
+      try {
+        setStatus(loadingStatus.LOADING)
+        const response = await axios.get(
+          `http://localhost:8000/api/plan/${planIdParam}`,
```

Comment:
add a note to remove the hardcoded localhost component - this won't work when we've deployed the app

Reply:
added note